reposerver/repository/repository_test.go

```diff
@@ -3072,6 +3127,223 @@ func TestGetGitFiles(t *testing.T) {
 	assert.Equal(t, expected, fileResponse.GetMap())
 }
 
+func TestErrorCompareRevisions(t *testing.T) {
+	type fields struct {
+		service *Service
+	}
+	type args struct {
+		ctx     context.Context
+		request *apiclient.CompareRevisionsRequest
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *apiclient.CompareRevisionsResponse
+		wantErr assert.ErrorAssertionFunc
+	}{
+		{name: "InvalidRepo", fields: fields{service: newService(".")}, args: args{
+			ctx: context.TODO(),
+			request: &apiclient.CompareRevisionsRequest{
+				Repo:           nil,
+				Revision:       "HEAD",
+				SyncedRevision: "sadfsadf",
+			},
+		}, want: nil, wantErr: assert.Error},
```

Comment:
Small thing, but I think it is easier to read when each args is defined on its own line.

```suggestion
		{
			name: "InvalidRepo", 
			fields: fields{service: newService(".")}, 
			args: args{
				ctx: context.TODO(),
				request: &apiclient.CompareRevisionsRequest{
					Repo:           nil,
					Revision:       "HEAD",
					SyncedRevision: "sadfsadf",
				},
			}, 
			wantErr: assert.Error
		},
```

Reply:
Agreed, but that's how the rest of the file is formatted. I'd rather stay consistent